smalltalksrc/VMMakerTests/VMObjectLayoutTests.class.st

```diff
@@ -61,6 +61,18 @@ VMObjectLayoutTests >> testHeaderOfObjectEncodesTheCorrectAmountOfSlots [
 			equals: objSize ]
 ]
 
+{ #category : #'tests - integerValues' }
+VMObjectLayoutTests >> testNegativeIntegerValuesInRange [
+
```

Comment:
Why only -17?
Is is a magic number?
May be we could have -2, 0, -100
What about the smallest minimal negative integer?